atomics/T1115/T1115.yaml

```diff
@@ -26,4 +26,13 @@ atomic_tests:
       echo Get-Process | clip
       Get-Clipboard | iex
     name: powershell
+- name: Execute commands from clipboard
+  description: Echo a command to clipboard and execute it
+  supported_platforms:
+  - macos
+  executor:
+    command: |-
+      echo ifconfig | pbcopy
```

Comment:
Did you want the literal string `ifconfig` to be in the clipboard? Or did you want the output of the `ifconfig` command in the clipboard?

If the latter you want `ifconfig | pbcopy`

Reply:
Thanks @rc-grey. I want the literal string ifconfig in the clipboard. Lets say attacker tries to execute commands in the clipboard.

Reply:
Doh - my brain was mush - completely read that wrong yesterday. Looks good